opencanada/settings/base.py

```diff
@@ -63,6 +63,8 @@ def get_env_variable(var_name):
     'taggit',
     'modelcluster',
 
+    'core',
```

Comment:
Moved core above wagtail apps so that I can override the templates.  Django will use the first template with the matching name according to the order of the INSTALLED_APPS.